src/redux/categories/categories.js

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+// eslint-disable-next-line import/no-extraneous-dependencies
```

Comment:
- [ ] Please don't disable eslint rules. There are here to help us write best code possible.